frontend/src/metabase/metadata/components/DiscardFieldValuesButton/DiscardFieldValuesButton.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useRef, useState } from "react";
+import { t } from "ttag";
+
+import { useDiscardFieldValuesMutation } from "metabase/api";
+import { useDispatch } from "metabase/lib/redux";
+import { addUndo } from "metabase/redux/undo";
+import { Button } from "metabase/ui";
+import type { FieldId } from "metabase-types/api";
+
+interface Props {
+  fieldId: FieldId;
+}
+
+export const DiscardFieldValuesButton = ({ fieldId }: Props) => {
+  const dispatch = useDispatch();
+
+  const [started, setStarted] = useState(false);
+  const timeoutIdRef = useRef<number>();
+  const [discardFieldValues, { error }] = useDiscardFieldValuesMutation();
+
+  const handleClick = async () => {
```

Comment:
Should a click here trigger a state change to show that the request has is in progress?